Create README.md (#7)
BredOS/sbc-pkgbuilds: README.md

# sbc-pkgbuilds 

> This repository contains Arch Linux PKGBUILDs to build kernel packages for SBCs with command `makepkgs`. The PKGBUILDs define how to build the packages, including dependencies, build steps, and installation steps.

Reviewer: ```suggestion
This repository contains Arch Linux ARM PKGBUILDs to build kernel packages for SBCs.
The PKGBUILDs define how to build the packages, including dependencies, build steps, and installation steps.
```

> The build will grab the kernel source code from the [BredOS kernel repository]() and build the archlinux kernel package, which can be then incorporated into the BredOS image in the [BredOS images repository](https://github.com/BredOS/images).

Reviewer: It's better off not being included at all, since we use lots of different sources for different boards.
```suggestion
```

> It won't provide the complete image to be burned into the SBCs, but only the kernel archlinux package.

Reviewer: ```suggestion
```

> Please refer to [BredOS images repository](https://github.com/BredOS/images) about how to build the complete image to use in various SBCs.

Reviewer: ```suggestion
Please refer to the [BredOS images repository](https://github.com/BredOS/images) for image build instructions.
```

## Usage

> To build the packages, clone this repository and run `makepkg` in the directory of the package you want to build. 

Reviewer: ```suggestion
To build a package from this repository clone it and then run `makepkg -s` in the directory of the package you want to build.
```

> Steps:

Reviewer: ```suggestion
## `linux-rockchip-rkr3` modifications
To apply patches to the the `linux-rockchip-rkr3` kernel:
```

> - Run `makepkg -sr` to pull the kernel source code from the [BredOS kernel repository](https://github.com/BredOS/linux-rockchip), which is saved in `linux-rockchip-rkr3/src/linux-rockchip`.

Reviewer: ```suggestion
- Run `makepkg -sr` to pull the kernel source code from the [BredOS rockchip kernel fork](https://github.com/BredOS/linux-rockchip) (rk6.1-rkr3 branch), which is saved in `linux-rockchip-rkr3/src/linux-rockchip`.
```

- Make the necessary changes in `linux-rockchip-rkr3/src/linux-rockchip`.

> - Note that your changes won't get saved if you remove the directory.

Reviewer: If the user does not know about git nothing here will matter either way.
```suggestion
```

> - After making the changes, run `makepkg -srf` to make the kernel package in arch format.

Reviewer: ```suggestion
- After making the changes, run `makepkg -srf` to build and package the kernel.
```

> Specfic examples:
> - linux-rockchip-rkr3: used in orange pi 5

Reviewer: Just an outliar example isn't worth including.
```suggestion
```
